src/pages/partnerships.svelte

```diff
@@ -0,0 +1,58 @@
+<Meta title={ $_('title.services') } description={ $_('description.services') }/>
```

Comment:
I think a title should be "Partnerships".

Reply:
changed.